mackup/applications/sublime-text-3.cfg

```diff
@@ -2,6 +2,7 @@
 name = Sublime Text 3
 
 [configuration_files]
-Library/Application Support/Sublime Text 3/Packages/User
```

Comment:
Yes, IMO this change is unnecessary. I have verified that the current configuration works (had to set up a new laptop just yesterday) so I don't see any reason to change it. Perhaps the source of confusion - and why some folks think this is not working - is that the Sublime Package Control package needs to be installed manually. Once that is done, it reads some configuration files and pulls the other packages automatically. If we sync Packages and Installed Packages, probably that manual step would not be required, but given the author (of Package Control) advises against it I'm not sure if mackup should work that way.